docker/ravendb-ubuntu1604/run-raven.sh

```diff
@@ -1,137 +1,9 @@
 #!/bin/bash
-
 COMMAND="./Raven.Server"
-
-CONFIG_FILE="settings.json"
-if [ -f "$CUSTOM_CONFIG_FILE" ]; then
-    CONFIG_FILE="$CUSTOM_CONFIG_FILE"
-fi
-
-FIRST_RUN=0
-if [ ! -f "firstrun" ]; then
-    FIRST_RUN=1
-    touch firstrun
-fi
-
-SERVER_URL_SCHEME="http"
-
-if [ ! -z "$CERTIFICATE_PATH" ]; then
-    SERVER_URL_SCHEME="https"
-else
-    if [ "$FIRST_RUN" -eq "0" ]; then
-        # not first run
-
-        SETUP_MODE_SECURED=$(jq '.Setup.Mode, ."Setup.Mode"' "$CONFIG_FILE" | grep -E 'Secured|LetsEncrypt')
-        
-        if [ ! -z "$SETUP_MODE_SECURED" ]; then
-            SERVER_URL_SCHEME="https"
-        fi
-    fi
-fi
-
-COMMAND="$COMMAND --ServerUrl=$SERVER_URL_SCHEME://0.0.0.0:8080"
-COMMAND="$COMMAND --ServerUrl.Tcp=tcp://0.0.0.0:38888"
-
-NO_SETUP=0
-
-if [ ! -z "$PUBLIC_SERVER_URL" ]; then
-    COMMAND="$COMMAND --PublicServerUrl=$PUBLIC_SERVER_URL"
-    NO_SETUP=1
+export RAVEN_ServerUrl="http://$(hostname):8080"
```

Comment:
What about the https case?

Reply:
no http case. this is internal default which can be override by settings.json (you can add there https)

Reply:
Note that this is intended specifically for the setup wizard mode, which will override everything else.